tasks/create-cert-webroot.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Check if certificate already exists.
+  stat:
+    path: /etc/letsencrypt/live/{{ cert_item.domains | first }}/cert.pem
+  register: letsencrypt_cert
+
+- name: Create deploy hook
+  copy:
+    content: "{{ certbot_deployhook }}"
+    dest: /etc/letsencrypt/renewal-hooks/deploy/ansible.sh
+    mode: u+rwx
+  run_once: yes
+  when: certbot_deployhook is defined
+
+- debug:
```

Comment:
Left over debug task?

Reply:
@rfaircloth-splunk Would indeed be nice to get this merged some time soon!

Reply:
Removed the debug statements